src/mappings/rewardsDistributor.ts

```diff
@@ -40,7 +42,7 @@ export async function handleRewardsDistributed(
   );
   assert(event.args, 'No event args');
 
-  const { indexer, eraIdx } = event.args;
+  const { indexer, eraIdx, rewards: eventRewards } = event.args;
```

Comment:
You can get commission from this event, which you will need to create commission part of EraReward